rest/bootstrap_test.go

```diff
@@ -139,6 +137,8 @@ func bootstrapStartupConfigForTest(t *testing.T) StartupConfig {
 	config.Bootstrap.Server = base.UnitTestUrl()
 	config.Bootstrap.Username = base.TestClusterUsername()
 	config.Bootstrap.Password = base.TestClusterPassword()
+	config.Bootstrap.ServerTLSSkipVerify = base.BoolPtr(true)
```

Comment:
Could probably do a similar to thing as below. I think. Avoids setting TLS Skip Verify if we're not using TLS which is probably preferable to always setting it